src/lib/authui/main_dialog.rb

```diff
@@ -64,17 +65,18 @@ def dialog_content
                     PushButton(Id(:cancel), Label.FinishButton)
                 ]
             end
-            Left(HBox(
-                HWeight(10, Empty()),
-                HWeight(80, VBox(
-                    Left(Frame(_('System authentication and domain configuration'), VBox(
+            VBox(
+                Left(Heading(_(@heading_caption_untranslated))),
```

Comment:
This will not work, because the value of `heading_caption_untranslated` will not be found in the message catalog. That is because the string is not enclosed with `_(...)`